cmd/gb-vendor/fetch.go

```diff
@@ -32,7 +34,7 @@ func addFetchFlags(fs *flag.FlagSet) {
 	fs.StringVar(&branch, "branch", "", "branch of the package")
 	fs.StringVar(&revision, "revision", "", "revision of the package")
 	fs.StringVar(&tag, "tag", "", "tag of the package")
-	fs.BoolVar(&recurse, "no-recuse", true, "do not fetch recursively")
+	fs.BoolVar(&noRecurse, "no-recurse", true, "do not fetch recursively")
```

Comment:
Ok you're right, just wanted to check the flagset type in package flag to make sure I wasn't crazy. So that means that to enable non-recursive fetch the user will have to do:

    -no-recurse=true

Because the default is false so if no value is specified from the cli the flag defaults to false (once I make the update), am I interpreting that correctly?